packages/hyperion-autologging/src/AutoLogging.ts

```diff
@@ -151,3 +153,25 @@ export function getSurfaceRenderer(defaultALSurfaceHOC?: ALSurface.ALSurfaceHOC)
   );
   return renderer;
 }
+
+
+/**
+ * Gets the init options passed when initializing AutoLogging.
+ * Can be useful to get configured channels, registered events, and other information after framework initialization.
+ */
+export function getInitOptions(): InitOptions {
```

Comment:
It looks like you mainly need the `channel` from the init options. 
Another , perhaps more reliable approach is to do something similar to the following where the application explicitly passes the channel to the component. 
https://github.com/facebook/hyperion/blob/main/packages/hyperion-autologging-visualizer/src/component/ElementTextTooltip.react.tsx

Effectively, we just need to call the following from the target application that prepares the visualizer. 
https://github.com/facebook/hyperion/blob/main/packages/hyperion-autologging-visualizer/src/index.ts

Reply:
It may also be useful to know which events are being emitted on the channel (e.g. UIPublisher registered events) so that the visualizer can itself set up listeners for those events,  but yes mostly just need channel.